projects/1.20/assets/domum-ornamentum/domum_ornamentum/lang/zh_cn.json

```
@@ -0,0 +1,210 @@
+  "domum_ornamentum.extra.name.format.gray": "灰色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.green": "绿色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.light_blue": "淡蓝色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.light_gray": "淡灰色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.lime": "黄绿色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.magenta": "品红色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.orange": "橙色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.pink": "粉红色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.purple": "紫色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.red": "红色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.white": "白色%s附加装饰方块",
+  "domum_ornamentum.extra.name.format.yellow": "黄色%s附加装饰方块",
+  "domum_ornamentum.fancydoor.center.block.format": "  - 材料：%s",
+  "domum_ornamentum.fancydoor.center.header": "中心：",
+  "domum_ornamentum.fancydoor.frame.block.format": "  - 材料：%s",
+  "domum_ornamentum.fancydoor.frame.header": "边框：",
+  "domum_ornamentum.fancydoor.name.format": "精致%s门",
+  "domum_ornamentum.fancydoor.type.format": "变种：%s",
+  "domum_ornamentum.fancydoor.type.name.creeper": "苦力怕",
+  "domum_ornamentum.fancydoor.type.name.full": "填满",
+  "domum_ornamentum.fancytrapdoor.center.block.format": "  - 材料：%s",
+  "domum_ornamentum.fancytrapdoor.center.header": "中心：",
+  "domum_ornamentum.fancytrapdoor.frame.block.format": "  - 材料：%s",
+  "domum_ornamentum.fancytrapdoor.frame.header": "边框：",
+  "domum_ornamentum.fancytrapdoor.name.format": "精致%s活板门",
+  "domum_ornamentum.fancytrapdoor.type.format": "变种：%s",
+  "domum_ornamentum.fancytrapdoor.type.name.creeper": "苦力怕",
+  "domum_ornamentum.fancytrapdoor.type.name.full": "填满",
+  "domum_ornamentum.fence-gate.name.format": "%s栅栏门",
+  "domum_ornamentum.fence.name.format": "%s栅栏",
+  "domum_ornamentum.group": "类别：",
+  "domum_ornamentum.light.center.block.format": "  - 材料：%s",
+  "domum_ornamentum.light.center.header": "中心：",
+  "domum_ornamentum.light.frame.block.format": "  - 材料：%s",
+  "domum_ornamentum.light.frame.header": "边框：",
+  "domum_ornamentum.light.frame.name.format": "镶边%s",
+  "domum_ornamentum.light.frame.type.center_light": "中心",
+  "domum_ornamentum.light.frame.type.crossed_light": "十字",
+  "domum_ornamentum.light.frame.type.fancy_light": "精致",
+  "domum_ornamentum.light.frame.type.format": "  - 样式：%s",
+  "domum_ornamentum.light.frame.type.four_light": "四边",
+  "domum_ornamentum.light.frame.type.framed_light": "镶边",
+  "domum_ornamentum.light.frame.type.horizontal_light": "水平",
+  "domum_ornamentum.light.frame.type.vertical_light": "垂直",
+  "domum_ornamentum.light_brick.name.format": "亮纹%s砖块",
+  "domum_ornamentum.light_brick_stair.name.format": "亮纹%s砖楼梯",
+  "domum_ornamentum.origin.tooltip": "由建筑师切割机合成",
+  "domum_ornamentum.panel.block.format": "材料：%s",
+  "domum_ornamentum.panel.name.format": "%s面板",
+  "domum_ornamentum.panel.type.format": "变种：%s",
+  "domum_ornamentum.panel.type.name.boss": "浮雕",

```

```suggestion
  "domum_ornamentum.panel.type.name.boss": "突出",
```
在游戏中是某种中间盖子突出的活板门，译作浮雕不妥。
然后下面也记得改。然后我approve

---

没关系我也是偶尔看一眼哈哈哈